lib/daru/io/importers/xlsx.rb

```diff
@@ -0,0 +1,96 @@
+require 'daru/io/importers/base'
+
+module Daru
+  module IO
+    module Importers
+      class XLSX < Base
+        Daru::DataFrame.register_io_module :from_xlsx, self
+
```

Comment:
Is the XLSX Importer intended be used directly as `Daru::DataFrame.from_xlsx(...)`, or to be redirected from the Excel Importer if filename ends with `.xlsx`?